client/src/components/pages/admin/Submitted_Id/index.js

```diff
@@ -1,5 +1,406 @@
+                            variant="body2"
+                            color="primary"
+                            align="left"
+                          >
+                            Submitted
+                          </Typography>
+                        </td>
+                        <td />
+                      </tr>
+                      <tr>
+                        <td className="item">
+                          <Typography variant="body2" align="left">
+                            codewars Score
+                          </Typography>
+                        </td>
+                        <td className="points-value">
+                          <Typography
+                            variant="body2"
+                            color="primary"
+                            align="left"
+                          >
+                            {data.codeWarsKyu}
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td className="item">
+                          <Typography variant="body2" align="left">
+                            freecodecamp Score
+                          </Typography>
+                        </td>
+                        <td className="points-value">
+                          <Typography
+                            variant="body2"
+                            color="primary"
+                            align="left"
+                          >
+                            {data.freeCodeCampPoints}
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td className="item" colSpan="2">
+                          <Typography variant="body2" align="left">
+                            <Checkbox
+                              checked={data && data.freeCodeCampTopics}
+                              className="profile_check_box"
+                              color="primary"
+                              disabled
+                            />
+                            FreeCode Camp Topics
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td className="item" colSpan="2">
+                          <Typography variant="body2" align="left">
+                            <Checkbox
+                              checked={data && data.applicationSubmittedDate}
+                              className="profile_check_box"
+                              color="primary"
+                              disabled
+                            />
+                            Submitted
+                            {data.applicationSubmittedDate &&
+                              `on ${
+                                data.applicationSubmittedDate.split('T')[0]
+                              }`}
```

Comment:
add space before the word on to be displayed submitted on not submittedon 
```suggestion
                            {data.applicationSubmittedDate &&
                              `  on ${
                                data.applicationSubmittedDate.split('T')[0]
                              }`}
```